### DISCUSSION TITLE
Only Sanctum vs Sanctum+Fortify

### DISCUSSION THREAD

**Discussion by celli33** [phpcfdi/api-descarga-masiva#12] [Ideas]
Me parece que dada la naturaleza del proyecto: tener un api que funcione sin la necesidad de un _frontend_.
Usar una autenticación _Sanctum_, via token, es el camino mas adecuado, ya que permitiría tener múltiples clientes: web, móvil, desktop, sin depender de las rutas nativas web.
Usando Fortify la autenticación del proyecto y por lo tanto del _frontend_, esta atada a la configuración _SPA_ de Sanctum y pondría más obstáculos a la hora de crear distintos clientes.
Por lo tanto propongo una autenticación por token via _Sanctum_.
¿Qué opinan?

**Comment by eclipxe13** [comment]
Debo leer un poco, por lo que había entendido con Fortify estaba ya implementado en server side las rutas estándar y la lógica. Mientras con sanctum habría que hacerlo. Y por lo que entiendo no son mutuamente excluyentes...

Voy a estudiarlo y opino después 🙂

**Reply by celli33** [reply]
Sí se pueden tener las dos cosas, las rutas estándar y aparte una auth por token solo sanctum. Pero creo que para el front end es más conveniente tener una sola forma y no las dos. Creo que nos puede dar más luz sobre esto @luffynando. ¿qué opinas?

**Comment by luffynando** [comment]
Es dependiendo del enfoque, si solo se le quiere dar acceso SPA, pues convendría aplicar sanctum +  fortify, pero si bien se le quiere dar multiplataforma, pues entonces si nomas sanctum vía solo tokens, así se permite usar una sola forma de autenticarse en múltiples clientes (web, desktop, movil, etc...)
